framework/python/src/test_orc/test_orchestrator.py

```diff
@@ -259,12 +262,20 @@ def _calculate_result(self):
             TestResult.COMPLIANT,
             TestResult.ERROR
           ]):
-        result = TestResult.NON_COMPLIANT
+        result = TestrunStatus.NON_COMPLIANT
 
       # Check Required if Applicable tests
       elif (test_result.required_result.lower() == "required if applicable"
             and test_result.result == TestResult.NON_COMPLIANT):
-        result = TestResult.NON_COMPLIANT
+        result = TestrunStatus.NON_COMPLIANT
+
+    # Change the result if pilot assessment used
+    if (self.get_session().get_target_device().test_pack ==
+        "Pilot Assessment"):
+      if result == TestrunStatus.COMPLIANT:
+        result = TestrunStatus.PROCEED
+      elif result == TestrunStatus.NON_COMPLIANT:
+        result = TestrunStatus.DO_NOT_PROCEED

```

Comment:
@jboddey Could you please add a new field to the response, like "pilot_recommendations," instead of replacing the result test? We want to display the information this way. https://screenshot.googleplex.com/79xfeL5q6F4fYjD